alphagov/paas-tech-docs
# Update to latest version of template

## What

- Revise highlight state for table of contents (https://github.com/alphagov/tech-docs-template/pull/93)
- Add footer (https://github.com/alphagov/tech-docs-template/pull/94)
- Tweak typography, mainly around relationship between headings and copy (https://github.com/alphagov/tech-docs-template/pull/95)
- Enable font smoothing in Firefox (https://github.com/alphagov/tech-docs-template/pull/95)
- Alter inline code styling (https://github.com/alphagov/tech-docs-template/pull/95)
- Simplify layout (https://github.com/alphagov/tech-docs-template/pull/99)

The equivalent diff in the template would be:
https://github.com/alphagov/tech-docs-template/compare/22f782e1519eba5d324698059e54b10a1168569e...master

## How to review

- Verify that files changed does not revert any modifications you may have made from the template
- Run `bundle exec middleman server`, check that everything looks fine on desktop and mobile breakpoints, and that there are e.g. no JS errors.

## Who can review

Any of the technical writers involved with the PaaS documentation or any of the PaaS team who are confident reviewing this.

I can talk people through the changes if useful.
